### DISCUSSION TITLE
请问是否会添加discord的适配器

### DISCUSSION THREAD

**Discussion by AlanHakua** [nonebot/discussions#159] [问答] [answered]
想在Discord频道里部署bot，但是没有看到相关的适配器，也没人讨论过关于discord适配的话题，想问一下是否有相关的开发计划

**Comment by yanyongyu** [comment, marked as answer]
目前人手不足哈:joy:

如果你想写适配器的话，欢迎在商店发布:tada:

**Reply by AlanHakua** [reply]
明白了，我回头看一下相关的开发文档，感谢您的回复！
